config/agents/app/agent/agentapp.yaml

```
@@ -60,3 +64,8 @@ spec:
             memory: 64Mi
       serviceAccountName: primaza-app-agent
       terminationGracePeriodSeconds: 10
+      volumes:
+      - name: primaza-secret-volume
+        secret:
+          defaultMode: 420
+          secretName: primaza-secret
```

the secret name is wrong

```suggestion
          secretName: primaza-svc-kubeconfig
```